docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,69 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** copies container images between hosts on a network. You can load to the remote host or from the remote host as well as in between two remote hosts.
+Note: `::` is used to specify the image name depending on if you are saving or loading.
+
+**podman image scp [GLOBAL OPTIONS]**
+
+**podman image** *scp [OPTIONS] NAME[:TAG] HOSTNAME[::FILELOCATION]*
+
+**podman image** *scp [OPTIONS] HOSTNAME[::IMAGENAME]*
```

Comment:
Shouldn’t this be
```suggestion
**podman image** *scp [OPTIONS] [HOSTNAME::]IMAGENAME*
```
? non-:: means a local image name, not a host.

Same for the other lines.

Reply:
hmmm I don't think so, if an image is specified after the colon it is on the host machine, this closely follows the format of `NAME[:TAG]`

Reply:
`name[:tag]` means:
- given `a`: `a` is a `name`; `[:tag]` is optional and not present, so `tag` is not set.
- given `a:b`: `a` is a `name` and `b` is a `tag`.

Following the same pattern, with `hostname[::imagename]`, `a` is a `hostname`. With `[hostname::]imagename`, `a` is an `imagename`.